docs/user-guide/src/irso/install-basics.md

```diff
@@ -0,0 +1,157 @@
+# Installing Ironic
+
+This document covers installing Ironic in different scenarios. You need to
+answer a few questions before you can pick the one that suits you:
+
+- Which physical network interface will be used for provisioning? Without any
+  configuration, Ironic will use the host cluster networking.
+
+- If you use a dedicated network interface, are you going to use the built-in
+  Keepalived service to manage the IP address on it? If not, you need to make
+  sure the interface has a usable address on each control plane node.
+
+- Do you want to use network boot (iPXE) during provisioning? DHCP adds more
+  requirements and requires explicit configuration. Without it, only virtual
+  media provisioning is possible (see [supported
+  hardware](../bmo/supported_hardware.md)).
+
+- Are you doing to use TLS for the Ironic API? It is not recommended to run
+  without TLS. To enable it, you need to manage the TLS secret. [Cert
+  Manager](https://cert-manager.io/) is the recommended service for it.
+
+## Using Ironic
+
+Regardless of the scenario you choose, you will need to create at least an
+`Ironic` object and wait for it to become ready:
+
+```bash
+NAMESPACE="scenario-N"  # change to match your deployment
```

Comment:
I don't quite get the point of having a different namespace for each scenario. Is it expected that people deploy all 3 together? If not, I would perhaps just leave it out.